instructor-training-process.md

```diff
@@ -0,0 +1,154 @@
+        [the certification repository][cert-repo]
+	and send it to her along with instructions for the items below.
+    1.  The instructions sent to the new instructor tell her to:
+        1.  join the `discuss` and `instructors` lists
+	1.  fill in a form on AMY to tell us where she is, what she's comfortable teaching, etc., and
+	1.  send us a short biography and photo to add to [our website][team-page].
+    1.  We manually add the biography and photo to the website
+        (since they almost always require reformatting).
+
+## Resources
+
+1.  [AMY][amy] is the web application we use to track instructors and workshops.
+    Most of it is only accessible to administrators,
+    but it does have a form where instructors can supply or update information about themselves.
+1.  There's a web form on AMY that people can use to apply for instructor training
+    (currently disabled).
+1.  We keep a list of trainees in a Google Docs spreadsheet with the following fields,
+    which we upate manually:
+    1.  name
+    1.  contact email
+    1.  which training course they took part in
+    1.  the name of the Software Carpentry lesson they're concentrating on (if any)
+    1.  the name of the Data Carpentry lesson they're concentrating on (if any)
+    1.  the URL of their Software Carpentry exercise submission (if any)
+    1.  whether or not they have submitted something for Data Carpentry
+    1.  the date and moderator of the discussion session they have taken part in
+        *   we currently store both values in one column
+	*   we do not record both a Software Carpentry and a Data Carpentry discussion session (though we should)
+    1.  the date on which they passed their Software Carpentry lesson demo (if any)
+	*   this date becomes the award date of the PDF certificate
+        *   we do not currently record the name of the person who was their instructor, or the name of the person who did their final test
+    1.  the date on which they passed their Data Carpentry lesson demo (if any)
+        *   the same caveats apply
+1.  Mentors offer lesson discussion sessions,
+    and trainees sign up for them,
+    on an Etherpad.
+1.  Trainers offer checkout sessions on a second Etherpad.
+    Trainees sign up for sessions on that Etherpad,
+    which we also presently use for taking notes during their sessions.
+1.  Discussion and checkout sessions are marked in Software Carpentry's community calendar
+    (a Google calendar).
+
+## Issues
+
+1.  As of Jan 2016,
+    the form for individuals to apply to take part in training has been disabled
+    while we try to clear our backlog (and come up with a better process).
+1.  We used a Google web form and a spreadsheet for group applications and sorted manually.
+    We are not systematically tracking whether groups run their promised workshops or not.
+1.  Negotiations with partners about instructor training sessions are tracked in a Google Docs spreadsheet.
+    Schedules are drawn up manually.
```

Comment:
1. They are separate spreadsheets.  We'd like to get rid of both of them.

2. The spreadsheet for individual applications has name, contact 
address, location, why they want to become an instructor.  The 
spreadsheet for groups has all of their names/email addresses, with the 
principal contact highlighted, plus the group's location and a 
description of what they plan to do once they're trained.

3. Moving data between the spreadsheets is clumsy and error-prone; 
synchronizing identities with AMY (e.g., figuring out which people in 
the queue for instructor training have been through workshops) is hard; 
spreadsheets don't send notifications on update.

4. I'm not allowed to do user interface design because I'm really, 
really bad at it :-)